clevertap-geofence/build.gradle

```diff
@@ -25,15 +25,14 @@ dependencies {
     testImplementation (libs.test.junit.platform.runner)
     testImplementation (libs.test.junit.jupiter.api)
 
-    testImplementation (libs.test.mockito.core)
+    testApi (libs.bundles.mockito)
```

Comment:
why did we change to api? the client will have to add it then? or is it pulling from core lib?

Reply:
Few unit tests were failing if testImplementation was used, now resolved